File: Buoi1/Big-O/vonglaplongnhau.jl
function printPairs(n)
    for i in 1:n         # Vòng lặp ngoài
        for j in 1:n     # Vòng lặp trong
            println("($i, $j)")  # In cặp (i, j)
        end
    end
end

function main()
    n = 3
    printPairs(n)  # Gọi hàm với n = 3
end

main()

File: Buoi1/Big-O/vonglaplongnhau.cpp
#include <iostream>
using namespace std;

void printPairs(int n) {
    for (int i = 1; i <= n; i++) {        // Vòng lặp ngoài
        for (int j = 1; j <= n; j++) {    // Vòng lặp trong
            cout << "(" << i << ", " << j << ")" << endl;  // In cặp (i, j)
        }
    }
}

int main() {
    int n = 3;
    printPairs(n);  // Output các cặp số từ (1,1) đến (n,n)
    return 0;
}
